Content.Server/StationEvents/Components/MassHallucinationsRuleComponent.cs

```diff
@@ -23,4 +24,7 @@ public sealed partial class MassHallucinationsRuleComponent : Component
 
     [DataField("sounds", required: true)]
     public SoundSpecifier Sounds = default!;
+
+    [DataField("affectedEntities"), ViewVariables(VVAccess.ReadOnly)]
```

Comment:
```suggestion
    [DataField, ViewVariables(VVAccess.ReadOnly)]
```
DataField uses the field name (with the first letter lowercased) by default, so you don't need to specify it.